packages/shared/lib/profile.ts

```diff
@@ -198,11 +199,10 @@ export const updateProfile = (path: string, value: ValuesOf<Profile> | ValuesOf<
  *
  * @returns {void}
  */
-export const cleanupInProgressProfiles = async (): Promise<void> => {
+export const cleanupInProgressProfiles = (): void => {
```

Comment:
How come we no longer want this to be async?

Reply:
We can put it back in; I simply removed it because there's no asynchronous code in [that function](https://github.com/iotaledger/firefly/blob/a1e554fc29b68b1314cde6358cab80cb1e1f7774/packages/shared/lib/profile.ts#L202).